main.go

```diff
@@ -29,6 +36,7 @@ func init() {
 	flag.BoolVar(&helpUsage, "help", false, "print help usage")
 	flag.StringVar(&environmentFilePath, "o", defaultEnvironmentFilePath, "environment file")
 	flag.BoolVar(&verboseOutput, "verbose", false, "enable verbose output")
+	flag.BoolVar(&requireDefaultRoute, "require-default-route", false, "make the script fail if a default route is not found")
```

Comment:
I woudl say we always want the default route, because without it the whole bootstrap procss most probabbly fail, so lets not make it optional

Reply:
Done